cms/djangoapps/contentstore/tests/test_tasks.py

```diff
@@ -307,17 +358,8 @@ def test_container_url_without_url_base_is_recognized_as_studio_url_scheme(self)
     def test_slash_url_without_url_base_is_recognized_as_studio_url_scheme(self):
         self.assertTrue(_is_studio_url(f'/static/test'))
 
-    @pytest.mark.parametrize("url, course_key, post_substitution_url",
-                             ["/static/anything_goes_here?raw", "1", "2"])
-    def test_url_substitution_on_static_prefixes(self, url, course_key, post_substitution_url):
-        with_substitution = _convert_to_standard_url(url, course_key)
-        assert with_substitution == post_substitution_url, f'{with_substitution} expected to be {post_substitution_url}'
-
-    def test_url_substitution_on_forward_slash_prefixes(self):
-        raise NotImplementedError
-
-    def test_url_subsitution_on_containers(self):
-        raise NotImplementedError
+    # TODO
```

Comment:
Could you remove the todo? If in the future, we found this test is valuable, we can add it.